org.civicrm.reqangular/src/common/services/api/contract.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/apis',
+  'common/services/api'
+], function (apis, _) {
+  'use strict';
+
+  apis.factory('api.contract', ['$log', 'api', function ($log, api) {
+    $log.debug('api.contract');
+
+    return api.extend({
+
+      /**
+       * This method returns all the contracts.
+       *
+       * It chains an additional call to the `getfulldetails` endpoint to also return
+       * full details on contracts
+       *
+       * @param  {Object} params  matches the api endpoint params (ex. contact_id)
+       * @return {Promise}
+       */
+      all: function (params) {
+        $log.debug('ContractAPI.all');
+
+        params = params || {};
+
+        params['api.HRJobContract.getfulldetails'] = {'jobcontract_id': '$value.id'};
+        params['deleted'] = 0;
```

Comment:
"deleted" is set to 0 as you are not supposed to get full details of a deleted contract.